lib/trento/domain/host/host.ex

```diff
@@ -183,6 +183,14 @@ defmodule Trento.Domain.Host do
     {:error, :host_not_registered}
   end
 
+  def execute(
```

Comment:
This has still some events on top, like `HostDetailsUpdated`.
Shouldn't we exclude these events as well?
I mean to updated something that "doesn't exist"